pulsar-metadata/src/main/java/org/apache/pulsar/metadata/impl/MetadataStoreFactoryImpl.java

```diff
@@ -81,4 +81,12 @@ public static String removeIdentifierFromMetadataURL(String metadataURL) {
         }
         return metadataURL;
     }
+
+    public static boolean isBasedOnZookeeper(String metadataURL) {
+        if (!metadataURL.contains("://")) {
```

Comment:
@merlimat it seems that metadata url doesn't contains `//` part?

From the codebase a metadata url to zk is `zk:127.0.0.1:2181` and a metadata url to memory is `memory:local`.